aws/resource_aws_elasticsearch_domain.go

```diff
@@ -358,10 +358,10 @@ func resourceAwsElasticSearchDomainUpdate(d *schema.ResourceData, meta interface
 
 	if err := setTagsElasticsearchService(conn, d, d.Id()); err != nil {
 		return err
-	} else {
-		d.SetPartial("tags")
 	}
 
+	d.SetPartial("tags")
+
```

Comment:
The reason is purely cosmetic. In case where there is an error, function will `return` from function so there is no need to keep `else`. From other side, by using 'else' we can show that there is a relation between `setTagsElasticsearchService` and `d.SetPartial("tags")`, so if there is any relation between them maybe we should keep `else` :>